mypy is failing on the stdlib

CI is currently failing on all PRs with the following message from mypy:

`stdlib/email/message.pyi:13: error: Cannot determine type of "Any" [has-type]`.

E.g.:

- https://github.com/python/typeshed/pull/7018
- https://github.com/python/typeshed/pull/7017

mypy appears to be failing on all platforms, but only on Python 3.8+.

I can also reproduce this error locally. No idea what's changed to make it suddenly start erroring as part of the CI checks, though.

--- a/stdlib/email/message.pyi
+++ b/stdlib/email/message.pyi
@@ -2,6 +2,7 @@ from email.charset import Charset
 from email.contentmanager import ContentManager
 from email.errors import MessageDefect
 from email.policy import Policy
+
 # using a type alias ("_HeaderType = Any") breaks mypy, who knows why
 from typing import Any, Any as _HeaderType, Generator, Iterator, Optional, Sequence, TypeVar, Union
 

--- a/stdlib/email/message.pyi
+++ b/stdlib/email/message.pyi
@@ -2,8 +2,8 @@ from email.charset import Charset
 from email.contentmanager import ContentManager
 from email.errors import MessageDefect
 from email.policy import Policy
-from typing import Any as _HeaderType  # using a type alias ("_HeaderType = Any") breaks mypy, who knows why
-from typing import Any, Generator, Iterator, Optional, Sequence, TypeVar, Union
+# using a type alias ("_HeaderType = Any") breaks mypy, who knows why
+from typing import Any, Any as _HeaderType, Generator, Iterator, Optional, Sequence, TypeVar, Union
 
 _T = TypeVar("_T")
 

--- a/stdlib/email/message.pyi
+++ b/stdlib/email/message.pyi
@@ -2,8 +2,16 @@ from email.charset import Charset
 from email.contentmanager import ContentManager
 from email.errors import MessageDefect
 from email.policy import Policy
-from typing import Any, Generator, Iterator, Optional, Sequence, TypeVar, Union
-from typing import Any as _HeaderType  # using a type alias ("_HeaderType = Any") breaks mypy, who knows why
+from typing import (  # using a type alias ("_HeaderType = Any") breaks mypy, who knows why
+    Any,
+    Any as _HeaderType,
+    Generator,
+    Iterator,
+    Optional,
+    Sequence,
+    TypeVar,
+    Union,
+)
 
 _T = TypeVar("_T")
 

--- a/stdlib/email/message.pyi
+++ b/stdlib/email/message.pyi
@@ -2,16 +2,8 @@ from email.charset import Charset
 from email.contentmanager import ContentManager
 from email.errors import MessageDefect
 from email.policy import Policy
-from typing import (
-    Any,
-    Any as _HeaderType,  # using a type alias ("_HeaderType = Any") breaks mypy, who knows why
-    Generator,
-    Iterator,
-    Optional,
-    Sequence,
-    TypeVar,
-    Union,
-)
+from typing import Any as _HeaderType  # using a type alias ("_HeaderType = Any") breaks mypy, who knows why
+from typing import Any, Generator, Iterator, Optional, Sequence, TypeVar, Union
 
 _T = TypeVar("_T")
 

--- a/stdlib/email/message.pyi
+++ b/stdlib/email/message.pyi
@@ -2,9 +2,9 @@ from email.charset import Charset
 from email.contentmanager import ContentManager
 from email.errors import MessageDefect
 from email.policy import Policy
-from typing import (  # using a type alias ("_HeaderType = Any") breaks mypy, who knows why
+from typing import (
     Any,
-    Any as _HeaderType,
+    Any as _HeaderType,  # using a type alias ("_HeaderType = Any") breaks mypy, who knows why
     Generator,
     Iterator,
     Optional,

--- a/stdlib/email/message.pyi
+++ b/stdlib/email/message.pyi
@@ -3,6 +3,7 @@ from email.contentmanager import ContentManager
 from email.errors import MessageDefect
 from email.policy import Policy
 from typing import Any, Generator, Iterator, Optional, Sequence, TypeVar, Union
+from typing import Any as _HeaderType  # using a type alias ("_HeaderType = Any") breaks mypy, who knows why
 
 _T = TypeVar("_T")
 
@@ -10,7 +11,6 @@ _PayloadType = Union[list[Message], str, bytes]
 _CharsetType = Union[Charset, str, None]
 _ParamsType = Union[str, None, tuple[str, Optional[str], str]]
 _ParamType = Union[str, tuple[Optional[str], Optional[str], str]]
-_HeaderType = Any
 
 class Message:
     policy: Policy  # undocumented

